src/Utils/util.ts

```diff
@@ -93,4 +94,15 @@ export function earlierDateOrTimeToday(timestamp: string): string {
     const dialogTime = moment(timestamp)
     const isDialogCreatedToday = dialogTime.diff(endOfYesterday) >= 0
     return dialogTime.format(isDialogCreatedToday ? 'LTS' : 'L')
+}
+
+export function isNewActionUnique(newAction: CLM.ActionBase, actions: CLM.ActionBase[]): boolean {
```

Comment:
Hmm, questioning putting this in utils given the context. If you're going to put this as a utility it needs to be as re-usable and generic as possible.

Would drop the "new" and rename as `isActionUnique`.  You don't know where this method will be used and can't guarantee it will be for "new" actions.  For example, we should probably also protect against any modification of an action that would make it not unique so in that case it would be `isEditedActionUnique` which doesn't make sense.

Reply:
Agree with the more generic name would be IsActionUnique.

Reply:
Hmm maybe my comments are not behaving the way I had intended. This was marked as resolved but I'm still seeing the old code.  Also `camelCase`